### PR TITLE
removed build errors coming due to commons io version update.

### PR DESCRIPTION
**Pull request by rgarg24**

### CODE REVIEW

File: projects/sitemanage/src/test/java/com/percussion/share/test/PSMatchers.java [hunk 1]
```diff
@@ -111,7 +111,7 @@ public boolean matchesSafely(String item)
                 {
                     stream = IOUtils.toInputStream(item, "UTF-8");
                 }
-                catch (IOException e)
```
Comment (natechadwick) [inline projects/sitemanage/src/test/java/com/percussion/share/test/PSMatchers.java:114]: @rgarg24  Why the change here?  We should be capturing named exceptions not general exceptions if possible.

Comment (rgarg24) [reply projects/sitemanage/src/test/java/com/percussion/share/test/PSMatchers.java:114]: @natechadwick yes Nate you are right. But in this case, IOException is never thrown due to which we were getting compile time error. That's why I made it general exception.

Comment (natechadwick) [reply projects/sitemanage/src/test/java/com/percussion/share/test/PSMatchers.java:114]: @rgarg24  Based on the JavaDoc they do not thro Exception https://javadoc.io/doc/commons-io/commons-io/2.14.0/org/apache/commons/io/IOUtils.html.  It looks like they throw [UnsupportedCharsetException](https://docs.oracle.com/javase/8/docs/api/java/nio/charset/UnsupportedCharsetException.html)

Comment (rgarg24) [reply projects/sitemanage/src/test/java/com/percussion/share/test/PSMatchers.java:114]: @natechadwick I have done the changes as suggested.

Comment (natechadwick) [reply projects/sitemanage/src/test/java/com/percussion/share/test/PSMatchers.java:114]: @rgarg24  The code should be catching the exception, not re-throwing it.

File: projects/sitemanage/src/test/java/com/percussion/share/test/PSMatchers.java [hunk 2]
```diff
@@ -104,14 +104,22 @@ public static Matcher<String> validXhtml() {
...
-                stream = IOUtils.toInputStream(item, "UTF-8");
+                try
+                {
+                    stream = IOUtils.toInputStream(item, "UTF-8");
+                }
+                catch (UnsupportedCharsetException e)
+                {
+                    throw new UnsupportedCharsetException("Exception in method matchesSafely(): "+e.getMessage());
+                }
                 boolean valid = validator.isValid(stream);
```
Comment (natechadwick) [inline projects/sitemanage/src/test/java/com/percussion/share/test/PSMatchers.java:124]: @rgarg24  I don't understand the return value here.  Shouldn't the method not throw the exception, log an error and return false?

Comment (rgarg24) [reply projects/sitemanage/src/test/java/com/percussion/share/test/PSMatchers.java:124]: @natechadwick Yes Nate you are right but since its a part of test classes, that's why throwing it so that it fails in the test case itself. The way it was written before.
If you think that we should just catch it and return false, then that also we can do.

Comment (rgarg24) [reply projects/sitemanage/src/test/java/com/percussion/share/test/PSMatchers.java:124]: @natechadwick I have done the changes as suggested.

**Review by natechadwick — APPROVED** [review]
